modules/efk-logging-configuring-image-about.adoc

```diff
@@ -9,15 +9,14 @@ Each cluster logging component uses an image.
 
 By default, cluster logging uses the following images:
 
-* *ELASTICSEARCH_IMAGE* deploys Elasticsearch, by default `docker.io/openshift/origin-logging-elasticsearch5:latest`.
-* *FLUENTD_IMAGE* deploys Fluentd, by default `docker.io/openshift/origin-logging-fluentd:latest`.
-* *KIBANA_IMAGE* deploys Kibana, by default `docker.io/openshift/origin-logging-kibana5:latest`.
-* *CURATOR_IMAGE* deploys Curator, by default `docker.io/openshift/origin-logging-curator5:latest`.
-* *OAUTH_PROXY_IMAGE* defines OAUTH for {product-title}, by default `docker.io/openshift/oauth-proxy:latest`.
-* *RSYSLOG_IMAGE* deploys Rsyslog, by default `docker.io/viaq/rsyslog:latest`.
+* *ELASTICSEARCH_IMAGE* deploys Elasticsearch, by default `quay.io/openshift/origin-logging-elasticsearch5:latest`.
+* *FLUENTD_IMAGE* deploys Fluentd, by default `quay.io/openshift/origin-logging-fluentd:latest`.
+* *KIBANA_IMAGE* deploys Kibana, by default `quay.io/openshift/origin-logging-kibana5:latest`.
+* *CURATOR_IMAGE* deploys Curator, by default `quay.io/openshift/origin-logging-curator5:latest`.
+* *OAUTH_PROXY_IMAGE* defines OAUTH for {product-title}, by default `quay.io/openshift/origin-oauth-proxy:latest`.
+//* *RSYSLOG_IMAGE* deploys Rsyslog, by default `docker.io/viaq/rsyslog:latest`.
```

Comment:
this has nothing to do with choosing to deploy Rsyslog. This is the image that will be used for RSYSLOG when it is deployed. A component being deployed or not is configured by the CR.

Same verbiage for all other IMAGES here.

Reply:
@ewolinetz Can the user change the rsyslog image by setting an env var outside the CR, or would we rather they not change it?

Reply:
these env vars are only available on the CLO deployment object. they are not available on a CR